src/themes/teams/components/Header/headerDescriptionStyles.ts

```diff
@@ -1,11 +1,17 @@
+import * as _ from 'lodash'
+
 import { pxToRem } from '../../utils'
-import { ICSSInJSStyle } from '../../../types'
+import { ICSSInJSStyle, ComponentSlotStylesInput } from '../../../types'
+import { HeaderProps } from '../../../../components/Header/Header'
+import { HeaderVariables } from './headerVariables'
 
-export default {
-  root: ({ variables: v }): ICSSInJSStyle => ({
+const headerStyles: ComponentSlotStylesInput<HeaderProps, HeaderVariables> = {
```

Comment:
`HeaderDescriptionVariables` should be used as a type here

Reply:
same applies to prop types, actually

Reply:
nice catch, fixed